### ISSUE TITLE
Add Francis Polignano as a contributer 

### ISSUE THREAD

**Issue by nicocrozier** [thenewboston-developers/Website#1156]
Add Francis Polignano as a contributer.

He created the DevOps, setting up the banks and nodes videos that were published last month. 

he is @FrancisPolignano on slack.

Thank You

**Comment by jamessspanggg** [comment]
@nicocrozier can you provide the contributor info with the following format?
```
Team: [TEAM NAME]

{
  "title": [JOB TITLE],
  "isLead": [true OR false],
  "payPerDay": 2800,
  "createdDate": "2020-07-08T02:14:59.307535Z",
  "contributor": {
    "contributorId": [CONTRIBUTOR ID],
    "displayName": [DISPLAY NAME],
    "githubUsername": [GITHUB USERNAME],
    "profileImage": [PROFILE IMAGE URL (Slack or GitHub URL)],
    "slackUsername": [SLACK USERNAME]
  }
}
```

**Comment by nicocrozier** [comment]
Team: [YouTube Educator]

{
  "title": [YouTube Educator],
  "isLead": [false],
  "payPerDay": 2800,
  "createdDate": "2020-07-08T02:14:59.307535Z",
  "contributor": {
    "contributorId": [CONTRIBUTOR ID],
    "displayName": [Francis Polignano],
    "githubUsername": [franchyze923],
    "profileImage": [https://ca.slack-edge.com/T07PJD1FZ-U01DQAFHKE3-aad7fcf8fa5b-512],
    "slackUsername": [@Francis Polignano]
  }
}

**Comment by singh-sarabjeet** [comment]
Fixed by #1164 
TNB account : c5778aee3cfd83c2088ae81c178c3d34931399f20c1492c33044d8cd85e21898
